src/test/java/nextstep/ladder/PointTest.java

```diff
@@ -1,48 +1,42 @@
 package nextstep.ladder;
 
 import nextstep.ladder.domain.Point;
-import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.*;
 import static org.junit.jupiter.api.Assertions.*;
 
-public class PointTest {
-    @Test
-    @DisplayName("사다리 첫 번째 Line 의 다음 값이 true 이면 가로 라인이 겹치지 않게 두번째 Line의 다음 값은 false이다.   ")
-    void init() {
-        assertThat(Point.init(true).insert(true).nextPoint()).isFalse();
-        assertThat(Point.init(true).insert(false).nextPoint()).isFalse();
-    }
+class PointTest {
 
     @Test
-    void insert() {
-        Point first = Point.init(false); // (false , false)
-        assertThat(first.move()).isEqualTo(0);
+    void create() {
+        assertAll(
+                () -> assertThat(Point.init(true).move()).isEqualTo(1),
+                () -> assertThat(Point.init(false).move()).isEqualTo(0)
+        );
     }
 
     @Test
-    void insert2() {
-        Point first = Point.init(true); // (false ,true)
-        assertThat(first.move()).isEqualTo(1);
+    void next() {
+        assertThatThrownBy(() -> Point.init(true).next(true));
```

Comment:
```suggestion
        Point point = Point.init(true);
        assertThatThrownBy(() -> point.next(true))
                .isInstanceOf(IllegalArgumentException.class);
```
현재는 init메서드에서 예외가 발생하는지, next 메서드에서 예외가 발생하는지 알 수 없어요 😢 
assertThatThrownBy 내부에는 정확히 예외 발생이 예측되는 곳을 넣어주세요
추가적으로 어떤 예외가 발생하는지도 명시하면 좋을 것 같습니다 ^_^